shopify-client/src/views/ProductCreate.vue

```diff
@@ -63,7 +63,7 @@ export default {
 }
 
 .center {
-  margin-left: 500px;
-  margin-right: 500px;
+  margin-left: 10%;
```

Comment:
Just a comment but % should not be used for margins. Different breakpoints should be used to specify the margins.